internal/core/src/common/FieldMeta.h

```diff
@@ -128,6 +139,11 @@ class FieldMeta {
         Assert(!is_vector());
     }
 
+    FieldMeta(const FieldName& name, FieldId id, DataType type, int64_t max_length_per_row)
+        : name_(name), id_(id), type_(type), string_info_(StringInfo{max_length_per_row}) {
```

Comment:
string_info_(max_length_per_row) -- can write in this way ?

Reply:
The type of string_info_ is std::optional<StringInfo> , and this way(string_info_(max_length_per_row)) not work